content/docs/getting-started/runtime.md

```diff
@@ -22,13 +22,38 @@ distributes load across multiple threads. The `threaded_scheduler` is the
 default for applications and the `basic_scheduler` is the default for tests.
 
 It's important to remember that all async tasks **must** be spawned on the
-runtime or no work will be performed.
+runtime or no work will be performed:
+
+```rust
+# #![allow(unused_must_use)]
+use tokio::net::TcpStream;
+
+#[tokio::main]
+async fn main() {
+  // Create a tcp stream, but do not call await.
+  TcpStream::connect("127.0.0.1:6142");
```

Comment:
If we are splitting hairs, it might be possible for some part of the connect process to happen without `.await`, but IMO this blurb is fine for getting started.